.github/workflows/backend.yml

```diff
@@ -56,6 +56,9 @@ jobs:
     needs: paths-filter
     if: ${{ (needs.paths-filter.outputs.not-ignore == 'true') || (github.event_name == 'push') }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11' ]
```

Comment:
Add `setup-java` to this job otherwise this matrix does nothing but only build the package twice on default jdk

Reply:
Done